docs/source/plotting/1DPlotsHelp.rst

```diff
@@ -88,22 +88,22 @@ Click the generate a script button |GenerateAScript.png| on a 1D Plot:
 .. code-block:: python
 
    # import mantid algorithms, numpy and matplotlib
-   from mantid.simpleapi import *
    import matplotlib.pyplot as plt
-   import numpy as np
-
+   from mantid.plots.utility import MantidAxType
    from mantid.api import AnalysisDataService as ADS
 
-   MAR11060 = ADS.retrieve('MAR11060') #May replace with Load('MAR11060')
+   MAR11060 = ADS.retrieve('MAR11060')
 
    fig, axes = plt.subplots(edgecolor='#ffffff', num='MAR11060-1', subplot_kw={'projection': 'mantid'})
-   axes.plot(MAR11060, color='#1f77b4', label='MAR11060: spec 1', specNum=1)
-   axes.plot(MAR11060, color='#ff7f0e', label='MAR11060: spec 2', specNum=2)
-   axes.plot(MAR11060, color='#2ca02c', label='MAR11060: spec 3', specNum=3)
+   axes.plot(MAR11060, color='#1f77b4', label='MAR11060: spec 1', wkspIndex=0)
+   axes.plot(MAR11060, color='#ff7f0e', label='MAR11060: spec 2', wkspIndex=1)
+   axes.plot(MAR11060, color='#2ca02c', label='MAR11060: spec 3', wkspIndex=2)
+   axes.tick_params(axis='x', which='major', **{'gridOn': False, 'tick1On': True, 'tick2On': False, 'label1On': True, 'label2On': False, 'size': 6, 'tickdir': 'out', 'width': 1})
+   axes.tick_params(axis='y', which='major', **{'gridOn': False, 'tick1On': True, 'tick2On': False, 'label1On': True, 'label2On': False, 'size': 6, 'tickdir': 'out', 'width': 1})
    axes.set_title('MAR11060')
-   axes.set_xlabel('Time-of-flight ($\mu s$)')
-   axes.set_ylabel('Counts ($\mu s$)$^{-1}$')
-   axes.legend().draggable()
+   axes.set_xlabel('Time-of-flight ($\\mu s$)')
+   axes.set_ylabel('Counts ($\\mu s$)$^{-1}$')
+   legend = axes.legend(fontsize=8.0).set_draggable(True).legend
```

Comment:
Just wondering the reason for creating a `legend` variable if it isn't used in these examples. Is it to show that `axes.legend()` will return the legend it creates?

Reply:
Good question - these script examples are supposed to be demonstrating what you get when you click the "generate script" button on a plot. The script that's generated when you do that seems to include the legend variable. I'm not sure why that is, but that's why it gets included in the example in the documentation.

Reply:
Okay, I think it's fine for it to be there, just curious.